python/lsst/meas/deblender/plugins.py

```diff
@@ -1011,9 +1010,9 @@ def makeTemplatesMonotonic(debResult, log):
 
 
 def clipFootprintsToNonzero(debResult, log):
-    """Clip non-zero spans in the template footprints for every peak in each filter.
+    r"""Clip non-zero spans in the template footprints for every peak in each filter.
```

Comment:
Remember that the [maximum line length is 79 characters](https://developer.lsst.io/python/style.html#docstring-and-comment-line-length-must-be-less-than-or-equal-to-79-columns); this line, and several others in this file, are too long. Please shorten them.